### ISSUE TITLE
Flip value can leak to other hosts when mixing agent and no-agent

### ISSUE THREAD

**Issue by arlake228** [perfsonar/psconfig#66]
This is an edge case, but ran into this while debugging an ESnet mesh. Let's say you have a group where the `addresses` of a mesh contain a mix of `no-agent` hosts and hosts with an agent. The same applies to a `disjoint` tests where one or both of `a-addresses` and `b-addresses` contain a fix of agent and no-agent. Any addresses after the `no-agent` in the list will have the `{% flip %}` template valuable set to true. This generally relates to latency(bg) tests where you can have the source side kick-off the test if the other end does not have a psconfig agent running, The result is that tests that otherwise would be kicked-off by the source host are instead kicked-off by the destination host. This often still works. 

So to recap to see this issue the following must be true:

* It must be a test using the `{% flip %}` template variable (probably a latencybg test)
* The group addresses must have a mix of agent and no-agent
* The no-agent is not the last address specified

This is an easy fix that was significantly shorter to implement than it was to describe it in this issue :)
